src/autogluon/bench/eval/evaluation/evaluate_results.py

```diff
@@ -102,6 +102,14 @@ def evaluate(
     if frameworks is None:
         frameworks = sorted(list(results_raw[FRAMEWORK].unique()))
     elif len(set(frameworks)) != len(frameworks):
+        d = dict()
+        for f in frameworks:
+            if f in d:
+                d[f] += 1
+            else:
+                d[f] = 1
```

Comment:
minor, I think this can be simplified with 
```
from collections import Counter
d = Counter(frameworks)
```

Reply:
Good point! Updated